### DISCUSSION TITLE
Combination between REST API and lambda.

### DISCUSSION THREAD

**Discussion by nickmadev** [aws/aws-cli#7412] [Q&A] [answered]
I want to upload images to S3 by using REST API.
I am not sure how can I trigger REST API to Lambda.
Lambda code is written using Node.js.
I passed the test in manual uploading to S3 in Lambda.

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
Hi @procoder333, thanks for reaching out. [This premium support documentation](https://aws.amazon.com/premiumsupport/knowledge-center/api-gateway-upload-image-s3/) should be what you're looking for; please let me know if this isn't right, or if you have any other questions.
